packages/server/graphql/private/types/SetIsFreeMeetingTemplateSuccess.ts

```diff
@@ -1,17 +1,15 @@
-import getRethink from '../../../database/rethinkDriver'
+import isValid from '../../isValid'
 import {SetIsFreeMeetingTemplateSuccessResolvers} from '../../private/resolverTypes'
 
 export type SetIsFreeMeetingTemplateSuccessSource = {
   updatedTemplateIds: string[]
 }
 
 const SetIsFreeMeetingTemplateSuccess: SetIsFreeMeetingTemplateSuccessResolvers = {
-  updatedTemplates: async ({updatedTemplateIds}) => {
-    const r = await getRethink()
-    const updatedTemplates = await r
-      .table('MeetingTemplate')
-      .getAll(r.args(updatedTemplateIds))
-      .run()
+  updatedTemplates: async ({updatedTemplateIds}, _args, {dataLoader}) => {
```

Comment:
this isn't a write, but i changed it anyways since we don't want to be reading directly from the DB in a mutation payload resolver.